tests/core/padding-utils/test_padding.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import unicode_literals
```

Comment:
Assumed it was for python 2/3 compatibility because it's imported in various places across the codebase.  Should I take it out?